webhook/resources/volume/mutator.go

```diff
@@ -106,6 +106,10 @@ func (v *volumeMutator) Create(request *admission.Request, newObj runtime.Object
 		patchOps = append(patchOps, fmt.Sprintf(`{"op": "replace", "path": "/spec/restoreVolumeRecurringJob", "value": "%s"}`, longhorn.RestoreVolumeRecurringJobDefault))
 	}
 
+	if volume.Spec.UnmapMarkSnapChainRemoved == "" {
+		patchOps = append(patchOps, `{"op": "replace", "path": "/spec/unmapMarkSnapChainRemoved", "value": "ignored"}`)
```

Comment:
Can use `UnmapMarkSnapChainRemovedIgnored` from v1beta2 volume instead of text `ignored` here